presto-spark-base/src/test/java/com/facebook/presto/spark/execution/TestNativeExecutionTask.java

```diff
@@ -96,26 +92,26 @@
 {
     private static final boolean TRACE_HTTP = false;
 
-    @Test(timeOut = 30000)
-    public void testRegular()
-            throws Exception
-    {
-        AtomicLong lastActivityNanos = new AtomicLong(System.nanoTime());
-        TestingTaskResource testingTaskResource = new TestingTaskResource(lastActivityNanos, FailureScenario.NO_FAILURE);
-
-        NativeExecutionTaskFactory nativeExecutionTaskFactory = createNativeExecutionTaskFactory(testingTaskResource);
-        NativeExecutionTask nativeExecutionTask = createNativeExecutionTask(nativeExecutionTaskFactory);
-
-        testingTaskResource.setInitialTaskInfo(nativeExecutionTask.getTaskInfo());
-        nativeExecutionTask.start();
-    }
-
-    @Test(timeOut = 30000)
-    public void testNoFailure()
-            throws Exception
-    {
-        runTest(FailureScenario.NO_FAILURE);
-    }
+//    @Test(timeOut = 30000)
```

Comment:
The right is just to add enabled=false flag in the Test. Commenting everything out is a bit ugly....

Reply:
Sounds good